Cargo.toml

```diff
@@ -13,7 +13,7 @@ exclude = ["resources/*.pcap"]
 
 [dependencies]
 askama = "0.10"
-chrono = "0.4"
+chrono = { version = "0.4", features = ["serde"] }
```

Comment:
this will mean that any ipc plugin built before this release will need a replacement? ie the rust lib and the ipc-plugin will need to be updated in lock step? or is the serialization used here compatible with the old method?

Eitherway, lets bump the version

Reply:
Nope. This is still the same version, just extra features. The IPC Plugin is actually sending over SystemTime and not chrono::DateTime, so it also won't be impacted.

We should bump the version anyways since the TLS structure is changing.

Reply:
Awesome!